### PR TITLE
Functional tests : Email subscription - Add some steps to check validation email

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Email subscription - Add some steps to check validation email
| Type?             | refacto
| Category?         | TE
| BC breaks?        |  no
| Deprecations?     | no
| How to test?      | CI and nightly are :green_apple: 
| UI Tests          | https://github.com/nesrineabdmouleh/testing_pr/actions/runs/7531663724
| Fixed issue or discussion?     | no
| Related PRs       | no
| Sponsor company   |

### CODE REVIEW

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Please don't remove `img/tmp/CACHEDIR.TAG`

File: img/tmp/CACHEDIR.TAG [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (Progi1984) [inline img/tmp/CACHEDIR.TAG]: It seems that it miss one line in file.

**Review by Progi1984 — COMMENTED** [review]
A last comment

File: img/tmp/CACHEDIR.TAG (hunk 1 above)
Comment (nesrineabdmouleh) [reply img/tmp/CACHEDIR.TAG]: Done

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**